mini3d-plus/shape.h

```diff
@@ -21,6 +21,7 @@ typedef struct
 	uint16_t p3;
 	uint16_t p4; // 0xffff if it's a tri
 	float colorBias;
+	int isDoubleSided : 1;
```

Comment:
I wonder if this should be `unsigned int` because a 1-bit integer can only represent 0 and 1 if it's unsigned? I have to check that. It still works, though.